patterns/hero-overlapped-book-cover-with-links.php

```diff
@@ -13,154 +13,81 @@
-	<span aria-hidden="true" class="wp-block-cover__background has-background-dim-100 has-background-dim has-background-gradient" style="background:linear-gradient(180deg,rgb(251,250,243) 51%,rgb(255,255,255) 51%)"></span>
-	<div class="wp-block-cover__inner-container">
-
-		<!-- wp:group {"align":"full","layout":{"type":"constrained","justifyContent":"center","contentSize":"1124px"}} -->
-		<div class="wp-block-group alignfull">
-
-			<!-- wp:columns {"verticalAlignment":"top","align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|80","left":"var:preset|spacing|80"}}}} -->
-			<div class="wp-block-columns alignwide are-vertically-aligned-top">
-
-				<!-- wp:column {"verticalAlignment":"top","width":"55%"} -->
-				<div class="wp-block-column is-vertically-aligned-top" style="flex-basis:55%">
-
-					<!-- wp:group {"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch","flexWrap":"nowrap","verticalAlignment":"space-between"}} -->
-					<div class="wp-block-group">
-
-						<!-- wp:group {"style":{"spacing":{"blockGap":"2rem"}},"layout":{"type":"constrained"}} -->
-						<div class="wp-block-group">
-
-							<!-- wp:paragraph {"metadata":{"name":"Headline"},"style":{"typography":{"fontStyle":"normal","fontWeight":"400","fontSize":"3rem"}}} -->
-							<p style="font-size:3rem;font-style:normal;font-weight:400">
+<div class="wp-block-cover alignfull has-contrast-color has-text-color has-link-color" style="padding-top:var(--wp--preset--spacing--80);padding-bottom:var(--wp--preset--spacing--80);min-height:818px"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-100 has-background-dim has-background-gradient" style="background:linear-gradient(180deg,rgb(251,250,243) 51%,rgb(255,255,255) 51%)"></span><div class="wp-block-cover__inner-container"><!-- wp:group {"align":"full","layout":{"type":"constrained","justifyContent":"center","contentSize":"1124px"}} -->
+		<div class="wp-block-group alignfull"><!-- wp:columns {"verticalAlignment":"top","align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|80","left":"var:preset|spacing|80"}}}} -->
+			<div class="wp-block-columns alignwide are-vertically-aligned-top"><!-- wp:column {"verticalAlignment":"top","width":"55%"} -->
+				<div class="wp-block-column is-vertically-aligned-top" style="flex-basis:55%"><!-- wp:group {"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch","flexWrap":"nowrap","verticalAlignment":"space-between"}} -->
+					<div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"2rem"}},"layout":{"type":"constrained"}} -->
+						<div class="wp-block-group"><!-- wp:heading {"metadata":{},"fontSize":"xx-large"} -->
+							<h2 class="wp-block-heading has-xx-large-font-size">
 								<?php echo esc_html_x( 'The Stories® Book', 'Hero - Overlapped book cover pattern headline text', 'twentytwentyfive' ); ?>
```

Comment:
Note that the registered trademark symbol in the heading needs to be removed: The Stories® Book.
See https://github.com/WordPress/twentytwentyfive/issues/180

Reply:
Symbol removed.